azurelinuxagent/common/protocol/hostplugin.py

```diff
@@ -56,8 +56,8 @@
 MAXIMUM_PAGEBLOB_PAGE_SIZE = 4 * 1024 * 1024  # Max page size: 4MB
 
 
-class HostPluginProtocol(object):  # pylint: disable=R0902
-    _is_default_channel = False
+class HostPluginProtocol(object): # pylint: disable=R0902
```

Comment:
This is removing a second space from before a comment.

Is this what we want? I know @narrieta had a giant PR a little while ago about _adding_ these in.

Reply:
I think this is the result of GitHub's automatic merge. Thanks, I'll make sure to update this and any other instance.